packages/teams-js/test/private/videoEx.spec.ts

```diff
@@ -6,6 +6,8 @@ import { errorNotSupportedOnPlatform, FrameContexts } from '../../src/public/con
 import { video } from '../../src/public/video';
 import { Utils } from '../utils';
 
+mockMediaStreamAPI();
```

Comment:
This isn't the right way to make a mock: it looks like after a quick scan of this function you're replacing defining window functions, but I don't see anywhere that you are undoing this at the end of these tests. Assuming I'm right, this means this unit test file will now be potentially causing problems in other unit test files. Please make sure that you have cleaned up all state including mocks at the end of your tests. Jest should have hooks you can use to run code at the beginning and end of unit test suites.

Reply:
added reset operation after test end